Cannot build

MacBookAir Sonoma 14.4.1
Node: v20.7.0
Npm: 10.2.3

`git clone https://github.com/SpyglassMC/Spyglass.git `
`cd Spyglass`
`npm i && npm run build`

Error:
```
npm WARN skipping integrity check for git dependency ssh://git@github.com/SPGoding/vanilla-datapack.git 

added 955 packages, and audited 979 packages in 26s

144 packages are looking for funding
  run `npm fund` for details

14 vulnerabilities (3 low, 4 moderate, 7 critical)

To address issues that do not require attention, run:
  npm audit fix

To address all issues (including breaking changes), run:
  npm audit fix --force

Run `npm audit` for details.

> @spyglassmc/root@1.0.0 build
> wireit

✅ [packages/locales:build] Already fresh
🏃 [build:packages] Running command "tsc -b packages"
✅ [build:packages] Executed successfully
🏃 [packages/vscode-extension:build] Running command "node esbuild.mjs prod"
🏃 [packages/playground:build] Running command "node esbuild.js prod"
Start building...
Start building...
Built successfully.
✅ [packages/playground:build] Executed successfully
✘ [ERROR] No loader is configured for ".node" files: ../../node_modules/fsevents/fsevents.node

    ../../node_modules/fsevents/fsevents.js:13:23:
      13 │ const Native = require("./fsevents.node");
         ╵                        ~~~~~~~~~~~~~~~~~

Error: Build failed with 1 error:
../../node_modules/fsevents/fsevents.js:13:23: ERROR: No loader is configured for ".node" files: ../../node_modules/fsevents/fsevents.node
    at failureErrorWithLog (/Users/ali/Desktop/Spyglass/Spyglass/node_modules/esbuild/lib/main.js:1636:15)
    at /Users/ali/Desktop/Spyglass/Spyglass/node_modules/esbuild/lib/main.js:1048:25
    at /Users/ali/Desktop/Spyglass/Spyglass/node_modules/esbuild/lib/main.js:993:52
    at buildResponseToResult (/Users/ali/Desktop/Spyglass/Spyglass/node_modules/esbuild/lib/main.js:1046:7)
    at /Users/ali/Desktop/Spyglass/Spyglass/node_modules/esbuild/lib/main.js:1075:16
    at responseCallbacks.<computed> (/Users/ali/Desktop/Spyglass/Spyglass/node_modules/esbuild/lib/main.js:697:9)
    at handleIncomingPacket (/Users/ali/Desktop/Spyglass/Spyglass/node_modules/esbuild/lib/main.js:752:9)
    at Socket.readFromStdout (/Users/ali/Desktop/Spyglass/Spyglass/node_modules/esbuild/lib/main.js:673:7)
    at Socket.emit (node:events:514:28)
    at addChunk (node:internal/streams/readable:343:12) {
  errors: [
    {
      detail: undefined,
      id: '',
      location: [Object],
      notes: [],
      pluginName: '',
      text: 'No loader is configured for ".node" files: ../../node_modules/fsevents/fsevents.node'
    }
  ],
  warnings: []
}
❌ [packages/vscode-extension:build] Failed with exit status 1
```

--- a/packages/playground/esbuild.js
+++ b/packages/playground/esbuild.js
@@ -16,7 +16,10 @@ try {
 		platform: 'node',
 		target: ['firefox89', 'chrome93'],
 		bundle: true,
-		external: ['electron', 'vscode'],
+		// fsevents is a MacOS specific native extension used by chokidar for file
+		// watching and should not be bundled:
+		// https://github.com/SpyglassMC/Spyglass/issues/1143
+		external: ['electron', 'fsevents', 'vscode'],
 		sourcemap: isDev,
 		minify: !isDev,
 	})

--- a/packages/vscode-extension/esbuild.mjs
+++ b/packages/vscode-extension/esbuild.mjs
@@ -17,7 +17,10 @@ try {
 		target: 'node16.13',
 		bundle: true,
 		outdir: './dist',
-		external: ['electron', 'vscode'],
+		// fsevents is a MacOS specific native extension used by chokidar for file
+		// watching and should not be bundled:
+		// https://github.com/SpyglassMC/Spyglass/issues/1143
+		external: ['electron', 'fsevents', 'vscode'],
 		sourcemap: isDev,
 		minify: !isDev,
 	})

